conjur/argument_parser/_hostfactory_parser.py

```diff
@@ -0,0 +1,122 @@
+    def _add_hostfactory_create(hostfactory_subparser: ArgparseWrapper):
+        hostfactory_create_name = 'create - Create token/s for hosts with restrictions'
+        hostfactory_create_usage = 'conjur [global options] hostfactory create <subcommand> [options] [args]'
+
+        hostfactory_create_subcommand_parser = hostfactory_subparser \
+            .add_parser(name="create",
+                        help='Creates a token or creates a host using a token',
+                        description=command_description(
+                            hostfactory_create_name, hostfactory_create_usage),
+                        epilog=command_epilog(
+                            'conjur hostfactory create token --hostfactoryid my_factory --cidr 10.10.1.2/31 '
+                            '--duration-days 2\t\t\t '
+                            'Create creates one or more identical tokens for hosts with restrictions\t\t',
+                            command='create',
+                            subcommands=['token']
+                        ),
+                        usage=argparse.SUPPRESS,
+                        add_help=False,
+                        formatter_class=formatter)
+        hostfactory_create = hostfactory_create_subcommand_parser.add_argument_group(
+            title=title_formatter("Options"))
+        hostfactory_create.add_argument('-h', '--help', action='help',
+                                        help='Display help screen and exit')
+        return hostfactory_create_subcommand_parser.add_subparsers(title="Subcommand", dest='action')
+
+    @staticmethod
+    def _add_hostfactory_create_token(menu: ArgparseWrapper):
+        hostfactory_create_token_name = 'token - Create token/s for hosts with restrictions'
+        hostfactory_create_token_usage = 'conjur [global options] hostfactory create token [options] [args]'
+
+        hostfactory_create_subcommand_parser = menu \
+            .add_parser(name="token",
+                        help=' Create token/s for hosts with restrictions',
+                        description=command_description(
+                            hostfactory_create_token_name, hostfactory_create_token_usage),
+                        epilog=command_epilog(
+                            'conjur hostfactory create token --hostfactoryid my_factory --cidr 10.10.1.2/31 '
+                            '--duration-days 2\t\t '
+                            'Creates one or more identical tokens for hosts with restrictions\t\t',
+                            command='token',
+                        ),
+                        usage=argparse.SUPPRESS,
+                        add_help=False,
+                        formatter_class=formatter)
+        # Options
+        hostfactory_create_token = hostfactory_create_subcommand_parser.add_argument_group(
+            title=title_formatter("Options"))
+        hostfactory_create_token.add_argument('-action_type', default='create_token', help=argparse.SUPPRESS)
```

Comment:
what is this?

Reply:
It helps diff create from revoke they both end in cli.handle_hostfactory_logic